doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,6 +169,72 @@ Tracing                                Jaeger
 
 **Table 5-7a:** Service examples.
 
+Platform Services - Load Balancer REquirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The tbale below specifies a set of requirements for the Load Balances platform service.
+
+.. list-table::
+   :widths: 16, 30, 30
+   :header-rows: 1
+
+* - Ref #
+  - Requirement
+  - Notes
+* - pas.lb.001
+  - The Load Balancer must support resource scaling
+  - 
+* - pas.lb.002
+  - The Load Balancer must support resource resiliency
+  - 
+* - pas.lb.003
+  - The Load Balancer must support scaling and resiliency in the local environment
+  - Define local environment: within a subnet, tenant network, Az of a cloud, ...
+* - pas.lb.004
+  - The Load Balancer should support resiliency in the global environment
+  - 
+* - pas.lb.005
+  - The Load Balancer must support either OSI Level 3/4 and/or Level 7 load-balancing.
```

Comment:
I would suggest to have L4 as mandatory and L7 as optional.
in Kubernetes,  Ingress controllers (L7) depends on a L4 LB anyway, so wouldn't exist without it as a standalone L7 LB entity

```suggestion
  - The Load Balancer must support OSI Level 3/4 load-balancing and may support Level 7 Load Balancing.
```